src/components/Like/LikeBtn.module.scss

```diff
@@ -23,11 +23,13 @@
 }
 
 .small {
-  height: 12px;
   stroke-width: 0;
-  margin-top: -2px;
   fill: var(--casper);
-  margin-right: 4px;
+
+  & .icon {
```

Comment:
CSS like this, without objective reason, should be avoided.
Amount of checks in js file didn't change meanwhile, here we have added a new conditional. `.small` should be applied directly to the `Icon`.